source/utilities.tex

```diff
@@ -15881,7 +15881,7 @@
   \tcode{is_constructible_v<T,}\br
   \tcode{Args...>} is \tcode{true} and the variable
   definition for \tcode{is_constructible}, as defined below, is known to call
-  no operation that is not trivial (~\ref{basic.types},~\ref{special}). &
+  no operation that is not trivial (\ref{basic.types},~\ref{special}). &
```

Comment:
Let me disagree here. LaTeX might very well reflow the entire paragraph (without causing an overfull hbox) to avoid a linebreak at the tilde. I think it's good practice not to have a cross-reference at the start of a line. Is there a way to tell LaTeX "I don't want you to linebreak here, but before you produce an Overfull hbox, do it nonetheless"? If we can't do that, we have to consider the few cases where ~\ref produces an Overfull hbox on an individual basis.